services/morph_chain/artifacts.mk

```diff
@@ -19,17 +19,33 @@ else
 endif
 
 # Download NeoFS ADM tool 
-get.adm: NEOFS_ADM_DEST=./vendor/neofs-adm
+get.adm: NEOFS_ADM_DEST=./vendor
+get.adm: NEOFS_ADM_BIN=neofs-adm
 get.adm: NEOFS_ADM_ARCHIVE=neofs-adm.tar.gz
 get.adm:
+	@mkdir -p ${NEOFS_ADM_DEST}
```

Comment:
It's a separate target for `NEOFS_ADM_DEST` then. We're misusing `make` enough already, additional script-like behavior better be avoided.